CONTRIBUTING.md

```diff
@@ -21,6 +21,7 @@ If you don't care how and why and just want something that works, follow these s
 1. `git remote set-url origin https://github.com/<your-github-username>/cli`
 1. `go get -u github.com/golang/dep/cmd/dep`
 1. `dep ensure`
+    * you may need to [install `dep`](https://github.com/golang/dep) if it's not already on your system
```

Comment:
Step 5 is actually the install step for `dep`, but it may not work for everyone's setup nor do we call it out as a dependency. With that I suggest we update step 5 include the link to install dep and update the dependencies section up top so that future contributors know it is a requirement for the development setup.

Reply:
Ah yeah, was wondering why that step was there. With my naive setup the command doesn't install/build the package, just fetches the project (as far as I can tell?). I'll fix this based on your requests.